package.json

```diff
@@ -1,46 +1,50 @@
 {
-    "name": "guide",
-    "private": true,
-    "author": "",
-    "engines": {
-        "node": "^16"
-    },
-    "scripts": {
-        "start": "webpack --mode=development --watch --progress",
-        "build": "webpack --mode=production"
-    },
-    "devDependencies": {
-        "@babel/cli": "^7.17.3",
-        "@babel/core": "^7.17.5",
-        "@babel/preset-env": "^7.16.11",
-        "babel-loader": "^8.2.3",
-        "copy-webpack-plugin": "^10.2.4",
-        "css-loader": "^6.6.0",
-        "file-loader": "^6.2.0",
-        "mini-css-extract-plugin": "^2.5.3",
-        "postcss": "^8.4.6",
-        "postcss-loader": "^6.2.1",
-        "postcss-preset-env": "^7.4.1",
-        "sass": "^1.49.8",
-        "sass-loader": "^12.6.0",
-        "webpack": "^5.69.1",
-        "webpack-cli": "^4.9.2",
-        "webpack-manifest-plugin": "^4.1.1"
-    },
-    "browserslist": [
-        "Firefox ESR",
-        "last 2 Chrome versions",
-        "last 2 ChromeAndroid versions",
-        "last 2 Edge versions",
-        "last 1 Firefox version",
-        "last 2 iOS versions",
-        "last 3 Safari versions",
-        "not safari 13",
-        "not ie 11"
-    ],
-    "dependencies": {
-        "@popperjs/core": "^2.11.5",
-        "bootstrap": "^5.1.3",
-        "lodash.debounce": "^4.0.8"
-    }
+  "name": "guide",
+  "private": true,
+  "author": "",
+  "engines": {
+    "node": "^16"
+  },
+  "scripts": {
+    "start": "webpack --mode=development --watch --progress",
+    "build": "webpack --mode=production",
+    "format": "prettier --write .",
+    "lint:format": "prettier --check .",
```

Comment:
Note I think this won’t process dotfiles (`.gitpod.yml`). If we wanted this we’d need a more complex pattern like the one we use in Wagtail.

Reply:
ahh ok - I always wondered why you did that.

Have raised https://github.com/wagtail/guide/issues/162 for an improvement.